javatests/dagger/internal/codegen/BUILD

```diff
@@ -37,6 +45,7 @@ GenJavaTests(
     plugins = ["//java/dagger/internal/codegen/bootstrap"],
     deps = [
         ":kotlin_injected_qualifier",
+        ":kotlin_object_with_member_injection",
```

Comment:
should these be folded into a single lib? naming was very specific so I went this way, just not sure what is preferred.

Reply:
This is fine for now, if it gets too big we can consider folding.